t/14-isotovideo.t

```diff
@@ -126,6 +127,36 @@ subtest 'upload assets on demand even in failed jobs' => sub {
     ok(!-e $pool_dir . '/base_state.json',        'no fatal error recorded');
 };
 
+# mock backend/driver
+{
+    package FakeBackendDriver;
+    sub new {
+        my ($class, $name) = @_;
+        my $self = bless({class => $class}, $class);
+        require "backend/$name.pm";
+        $self->{backend} = "backend::$name"->new();
+        return $self;
+    }
+    sub extract_assets {
+        my $self = shift;
+        $self->{backend}->do_extract_assets(@_);
+    }
+}
+
+subtest 'upload the asset even in an test incomplete job' => sub {
```

Comment:
"even in an test incomplete job"? Better: "even in an incomplete job", just strip the "test"